Resources/Private/Javascript/Helper/utilities.js

```diff
@@ -24,4 +24,8 @@ export const classList = (classes) => {
         .filter(entry => entry[1])
         .map(entry => entry[0])
         .join(' ');
+}
+
+export const fireEvent = (name) => {
+    document.dispatchEvent(new Event(name));
```

Comment:
Why not build a helper that can handle custom events as well?

Reply:
Done